packages/backend/src/modules/task/task.service.ts

```diff
@@ -0,0 +1,20 @@
+import { CreateTaskCommand } from 'shared'
+import _taskRepository from './task.repository'
+
+class TaskService {
+  // eslint-disable-next-line class-methods-use-this
+  createTask(createTaskCommand: CreateTaskCommand) {
+    console.log('Handling new task')
+    return _taskRepository.createTask(createTaskCommand)
```

Comment:
use here _boardRepository to change data from board

Reply:
Done